protostar/cheatable_starknet/cheaters/contracts.py

```diff
@@ -226,6 +226,15 @@ async def call(
             function_name=function_name,
             calldata=calldata,
         )
+
+        function_selector = get_selector_from_name(func_name=function_name)
```

Comment:
we usually use positional arg-style call when using this function:
```suggestion
        function_selector = get_selector_from_name(function_name)
```